### PR TITLE
Q4 tidymodels roundup post

### PR DESCRIPTION
**Pull request by juliasilge**
This post should be published we leave for the holidays, either late this week or first thing next week on Dec 20/21.

### CODE REVIEW

**Comment by juliasilge** [comment]
Preview is [available here](https://deploy-preview-551--tidyverse-org.netlify.app/blog/2021/12/tidymodels-2021-q4/)

**Comment by topepo** [comment]
No vetiver?

**Comment by juliasilge** [comment]
> No vetiver?

I don't think so, especially given that the CRAN version has next to nothing in it

File: content/blog/tidymodels-2021-q4/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+options(cli.width = 70, width = 70)
+```
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. 
+
+```{r}
+library(tidymodels)
+```
+
+Starting at the beginning of this year, we now publish [regular updates](https://www.tidyverse.org/categories/roundup/) here on the tidyverse blog summarizing what's new in the tidymodels ecosystem. You can check out the [`tidymodels` tag](https://www.tidyverse.org/tags/tidymodels/) to find all tidymodels blog posts here, including both our roundup posts as well as those that are more focused. The purpose of these quarterly posts is to share useful new features and any updates you may have missed.
```
Comment (hfrick) [inline content/blog/tidymodels-2021-q4/index.Rmd:48]:
```suggestion
Starting at the beginning of this year, we now publish [regular updates](https://www.tidyverse.org/categories/roundup/) here on the tidyverse blog summarizing what's new in the tidymodels ecosystem. You can check out the [`tidymodels` tag](https://www.tidyverse.org/tags/tidymodels/) to find all tidymodels blog posts here, including our roundup posts as well as those that are more focused. The purpose of these quarterly posts is to share useful new features and any updates you may have missed.
```

**Review by hfrick — APPROVED** [review]

**Review by EmilHvitfeldt — APPROVED** [review]
Looks good!

**Review by DavisVaughan — APPROVED** [review]

**Pull request merged**